borsh-derive/src/lib.rs

```diff
@@ -80,3 +118,38 @@ pub fn borsh_schema(input: TokenStream) -> TokenStream {
         Err(err) => err.to_compile_error(),
     })
 }
+
+#[proc_macro_attribute]
+pub fn borsh(args: TokenStream, input: TokenStream) -> TokenStream {
```

Comment:
Why is it needed? It used to work just fine without it. I guess your problem is that you used `#[borsh()]` before `#[derive()]` attribute, which we need to move after `#[derive()]` line anyway.